sirepo/uri_router.py

```diff
@@ -218,11 +224,74 @@ async def get(self):
         async def post(self):
             await self._route()
 
+    class _WebSocket(websocket.WebSocketHandler):
+        #        def on_close(self):
+        #            #TODO(robnagler): need to free resources
+        #            pass
+
+        async def on_message(self, msg):
+            self.msg_count += 1
+            pkdlog(
+                "start ws={} msg={} remote_ip={}",
+                self.ws_id,
+                self.msg_count,
+                self.remote_ip,
+            )
+            # TODO(robnagler) what if msg poorly constructed? Close socket?
+            c = pkjson.load_any(msg)
+            e, r, k = _path_to_route(c.uri[1:])
+            if e:
+                pkdlog("uri={} {}; route={} kwargs={} ", c.uri, e, r, k)
```

Comment:
Maybe add a pkdc controlled dump of the full message contents? I frequently use the network tab in the browser to see what payloads are going back and forth which no longer works with ws.